news/2022-10-09-changes-to-osu-mania-sr-and-pp.md

```diff
@@ -0,0 +1,61 @@
+
+Let's dive straight into what's new:
+
+## osu!mania changes
+
+### Long note difficulty changes
+
+A [new change](https://github.com/ppy/osu/pull/17913) in long note calculation fixes issues with fast releases giving disproportionately more star rating than they should.
+
+Before this change, all overlapping long notes would receive a fixed bonus. After this change, this bonus now scales proportionally with how long ago the last long note release was.
+
+This change also fixes an oversight with long notes in the rightmost column of a map taking precedence in overlap calculation.
+
+An in-depth [blog post](https://eve-ning.github.io/#/blog/004/) and [interactive demonstration](https://www.desmos.com/calculator/7eboanfzne) by [**Evening**](https://osu.ppy.sh/users/2193881) helps visualize how this change affects the long note bonus if you're interested.
+
+### Chord difficulty changes
+
+An [issue has been fixed](https://github.com/ppy/osu/pull/19120) where star rating would change depending on the order of notes in the beatmap file.
+
+This change makes sure that the difficulty of the entire chord is taken into account, rather than only the last note to be processed by the difficulty calculator.
+
+This primarily affects maps with heavy usage of chords (especially in higher key modes), for example:
+
+- [Cardboard Box - The Limit Does Not Exist [[7K] Infinity]](https://osu.ppy.sh/beatmapsets/379758#mania/831204): 7.45\* --> 7.71\*
+- [Manabu Namiki - Tenshi (True Final Boss) [[4K] MIPHATHER]](https://osu.ppy.sh/beatmapsets/1503789#mania/3080978): 5.36\* --> 5.44\*
+
+### Using accuracy instead of score for performance points
+
+Overall score has historically been used as the measure of skill a player demonstrates on a given map. With the new direction [osu!(lazer)](https://github.com/ppy/osu) is taking, it no longer makes sense to base performance entirely on this metric.
+
+With a [new change](https://github.com/ppy/osu/pull/18749), awarded pp will now be based on the accuracy the player gets on the map using this formula:
+
+![](/wiki/shared/news/2022-10-09-changes-to-osu-mania-sr-and-pp/accuracy-equation.png)
+
+At 80% accuracy and below, the player will receive no pp for their score. This scales linearly to 100% accuracy where the player receives the maximum possible pp for their play. Compared to the old system, this rewards high-accuracy plays with low score (e.g. 96% accuracy, 780k score) compared to lower-accuracy plays with high score (e.g. 94% accuracy, 830k score). However, some scores that previously received no pp will find that they now do reward some pp. The differences are visualised below, calculated using OD 8:
```

Comment:
```suggestion
This formula is almost exactly the same as how the actual accuracy you'd see on the results screen is computed. With this "accuracy" that's used in difficulty calculations, the only change is that MAX judgements are weighted more. See the [wiki article on accuracy](/wiki/Gameplay/Accuracy#osu!mania) if you'd like to compare.

Additionally, 80% accuracy and below, the player will receive no pp for their score. This scales linearly to 100% accuracy where the player receives the maximum possible pp for their play. Compared to the old system, this rewards high-accuracy plays with low score (e.g. 96% accuracy, 780k score) compared to lower-accuracy plays with high score (e.g. 94% accuracy, 830k score). However, some scores that previously received no pp will find that they now do reward some pp. The differences are visualised below, calculated using OD 8:
```

Reply:
Not sure about this one. It's similar, but a huge difference is that the MAX judgement actually contributes to accuracy with this formula, so it might be misleading. The lead in to the second paragraph also sounds weird.

Reply:
it *is* the exact same formula, except that MAX judgements have a weight of 320 instead of 300. i get your point though so i've edited the suggestion

Reply:
That reads much better.